app/models/training.rb

```diff
@@ -10,10 +10,23 @@ class Training < ApplicationRecord
   has_many :learning_modules, dependent: :destroy
   has_many :shifts, dependent: :nullify
   belongs_to :skill, optional: true
+  has_many :badge_templates
 
-  validates :name, presence: true, uniqueness: true
+  validates :name_en, presence: true, uniqueness: true
```

Comment:
Keep in mind, this column doesn't exist if the migrations never ran. And because there's migrations that run before this column is created, this line would cause an error because this column doesn't exist yet. A bit confusing, but that's why you keep DB actions atomic